server/db/recreate-schema.sql

```diff
@@ -1,10 +1,44 @@
 -- Drop tables in case they already exist
+
+DROP TABLE if exists work_experience;
+DROP TABLE if exists education;
+DROP TABLE if exists profiles;
```

Comment:
Can profiles and users table name be made singular as asked in the ticket acceptance criteria? Renaming users to user may need some change in the server code as well.